_general/installation/aptpackage.md

```diff
@@ -20,6 +20,13 @@ wget -O - https://deb.froxlor.org/froxlor.gpg | apt-key add -
 echo "deb https://deb.froxlor.org/debian $(lsb_release -sc) main" > /etc/apt/sources.list.d/froxlor.list
 ````
 
+### Debian 11
+```` shell
+apt-get -y install apt-transport-https lsb-release ca-certificates gnupg wget
+wget -O- https://deb.froxlor.org/froxlor.gpg | gpg --dearmor | tee /usr/share/keyrings/froxlor-archive-keyring.gpg
```

Comment:
if it's using the full path to the keyfile anyway, can't we just wget/curl the file? Why so complicated?

Reply:
example, this is how sury php repository does it now: https://packages.sury.org/php/README.txt